### PR TITLE
Comments fix

### PR DESCRIPTION
**Pull request by cvvergara**
Fixing comments on functions
- fix links
- fix names on the comments

@pgRouting/admins

### CODE REVIEW

File: sql/topology/analyzeOneway.sql [hunk 1]
```diff
@@ -37,47 +37,72 @@ THE SOFTWARE.
...
+   flipped segments. Basically if you count the edges coming into a node
+   and the edges exiting a node the number has to be greater than one.
+
+   * tab              - edge table name (TEXT)
+   * col              - oneway column name (TEXT)
+   * s_in_rules       - source node in rules
+   * s_out_rules      - source node out rules
+   * t_in_tules       - target node in rules
+   * t_out_rules      - target node out rules
+   * two_way_if_null  - flag to treat oneway nNULL values as by directional
```
[the remark this answers is not in the document]
Comment (cvvergara) [reply sql/topology/analyzeOneway.sql]: fixing this one

File: sql/topology/analyzeOneway.sql [hunk 2]
```diff
@@ -37,47 +37,72 @@ THE SOFTWARE.
...
+
+   After running this on a graph you can identify nodes with potential
+   problems with the following query.
 
 .. code-block:: sql
 
-       select pgr_analyzeGraph('mytab', 0.000002);
+       select * from vertices_tmp where in=0 or out=0;
```
[the remark this answers is not in the document]
Comment (cvvergara) [reply sql/topology/analyzeOneway.sql]: But not fixing this issue on this PR.

File: sql/topology/createverticestable.sql [hunk 3]
```diff
@@ -261,8 +261,8 @@ $BODY$
 -- COMMENTS
 
 
-COMMENT ON FUNCTION pgr_createverticestable(TEXT, TEXT, TEXT, TEXT, TEXT) 
-IS 'pgr_createverticestable
+COMMENT ON FUNCTION pgr_createverticestable(TEXT, TEXT, TEXT, TEXT, TEXT)
```
Comment (vicennial) [inline sql/topology/createverticestable.sql]: Shouldn't it be:
```suggestion
COMMENT ON FUNCTION pgr_createVerticesTable(TEXT, TEXT, TEXT, TEXT, TEXT)
```

File: sql/driving_distance/drivingDistance.sql [hunk 4]
```diff
@@ -95,7 +95,28 @@ ROWS 1000;
...
+   - Edges SQL with columns: id, source, target, cost [,reverse_cost]
+   - From ARRAY[vertices identifiers]
+   - Distance from vertices identifiers
+- Optional Parameters
+   - directed := true
+   - equicost := false
+- Documentation:
+   - ${PGROUTING_DOC_LINK}/pgr_drivingDistance.html
+';
+
```
Comment (vicennial) [inline sql/driving_distance/drivingDistance.sql]: Is it ok to have empty lines at the end of the document or it does not matter much?  
Some files have trailing empty lines and some do not.

Comment (dkastl) [reply sql/driving_distance/drivingDistance.sql]: Good point @vicennial !

Personally I think one empty last line is a good idea. I'm using `.editorconfig` to make my editor care about this, and there is such a file also in the pgRouting repository: https://github.com/pgRouting/pgrouting/blob/master/.editorconfig#L11

Probably everyone shoud work with an editor supporting this configuration setting.

Comment (cvvergara) [reply sql/driving_distance/drivingDistance.sql]: not enforcing that yet.

**Pull request merged**